test/suites/ecma6-tests/ecma6-destructuring-assignment-tests.js

```diff
@@ -0,0 +1,91 @@
+*/
+var test = QUnit.test;
+var skip = QUnit.skip;
+
+define(['pumascript'], function (puma) {
+	
+	QUnit.module("Ecma6-Destructuring-Assignment-Tests");
+	
+	skip("Array destructuring: assignment with array declaration", function(assert) {
+	var result = puma.evalPuma("let foo = ['one', 'two', 'three']; let [one, two, three] = foo; one === 'one' && two === 'two' && three === 'three';");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Array destructuring: assignment without array declaration", function(assert) {
+        var result = puma.evalPuma("let a, b; [a, b] = [1, 2]; a === 1 && b === 2;");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Array destructuring: variable order change", function(assert) {
+        var result = puma.evalPuma("let a = 1; let b = 3; [a, b] = [b, a]; b === 1 && a === 3");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Array destructuring: multiple return values", function(assert) {
+        var result = puma.evalPuma("function f() {return [1, 2];} let [a, b] = f(); a === 1 && b === 2;");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Array destructuring: multiple return values assigned to an array", function(assert) {
+        var result = puma.evalPuma("function f() {return [1, 2];} let a = f(); a[0] === 1 && a[1] === 2;");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Array destructuring: some return values ignored", function(assert) {
+        var result = puma.evalPuma("function f() {return [1, 2, 3];} let [a, , b] = f(); a === 1 && b === 3;");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Object destructuring: assignment with object declaration", function(assert) {
+	var result = puma.evalPuma("let o = {p: 42, q: true}; let {p, q} = o; p === 42 && q === true;");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Object destructuring: assignment with object declaration", function(assert) {
+	var result = puma.evalPuma("let o = {p: 42, q: true}; let {p: foo, q: bar} = o; foo === 42 && bar === true;");
+        assert.equal(result.value, true);
+	});	
+	
+	skip("Object destructuring: assignment without declaration", function(assert) {
+        var result = puma.evalPuma("let a, b; ({a, b} = {a:1, b:2}); a === 1 && b === 2;");
+        assert.equal(result.value, true);
+	});
+	
+	skip("Object destructuring: function arguments default, simple example", function(assert) {
+        var result = puma.evalPuma("let {x = 1, y = 200} = {x: 100};");
+        assert.equal(result.value, true);
+	});	
+	
+	skip("Object destructuring: function arguments default", function(assert) {
+        var result = puma.evalPuma("function drawES2015Chart({size = 'big', cords = { x: 0, y: 0 }, radius = 25} = {}){return [size, cords, radius]} let [size, cords, radius] = drawES2015Chart({cords: { x: 18, y: 30 }, radius: 30}); size === 'big' && cords.x === 18 && cords.y === 30 && radius === 30;");
```

Comment:
Please review the complexity of the test and try to reduce to the minimal expression.